dstep/translator/Output.d

```diff
@@ -1080,7 +1075,7 @@ class ClassData : StructData
         if (!(mangledName in mangledMethods))
         {
             mangledMethods.add(mangledName);
-            name = name.isBlank ? selector : name;
+            name = !name.length ? selector : name;
```

Comment:
You don't consistently check for emptiness. Sometimes explicitly checking for `length == 0` sometimes not.
You could selectively import `empty`.